style.css

```diff
@@ -894,14 +894,14 @@ ul {
   }
 
   .popupc {
-    width: fit-content;
-    height: max-content;
+    width: 75%;
+    height: 821px;
     flex-wrap: wrap;
     flex-direction: column;
-    justify-content: center;
+    /* justify-content: center; */
     align-items: center;
-    margin: 1px 14px;
-    left: 1px;
+    left: 45px;
+    margin: 0 145px;
```

Comment:
- [ ] You still have the same issues with your `popup` window. I think this is due to the element alignment and elements `position`. Please ensure that your `nav` menu is positioned relative to its parent container i.e if the button has as position `absolute` the parent's container should have `position: relative`.
- [ ] Also, make sure that the design desktop design is similar to the Figma template design.  In your case the alignment is too dissimilar, please take the time to resolve it. I know you can do it so go ahead and have good courage! 😊 

Your design

![image](https://user-images.githubusercontent.com/80612925/175646611-0657feb5-a148-42cb-9700-5d75ac500a5d.png)

The Figma design

![image](https://user-images.githubusercontent.com/80612925/175646718-6f760a12-47e4-4184-b231-a77139f21480.png)